configure does not check for required haskell modules

the configure script does not check for haskell modules old-time and temporary:
```bash
apt purge libghc-old-time-dev libghc-temporary-dev
./configure
make
```
which later cabal does, because of `cabal/ganeti.template.cabal`
```
Resolving dependencies...
Warning: solver failed to find a solution:
Could not resolve dependencies:
[__0] trying: ganeti-2.16 (user goal)
[__1] unknown package: temporary (dependency of ganeti)
[__1] fail (backjumping, conflict set: ganeti, temporary)
After searching the rest of the dependency tree exhaustively, these were the
goals I've had most trouble fulfilling: ganeti, temporary
Trying configure anyway.
Configuring ganeti-2.16...
cabal: Encountered missing dependencies:
old-time >=1.1.0.0, temporary >=1.1.2.3 && <1.4

make: *** [Makefile:4328: cabal_macros.h] Error 1
```

--- a/configure.ac
+++ b/configure.ac
@@ -692,6 +692,8 @@ AC_GHC_PKG_REQUIRE(cryptonite)
 AC_GHC_PKG_REQUIRE(lifted-base)
 AC_GHC_PKG_REQUIRE(lens)
 AC_GHC_PKG_REQUIRE(regex-pcre)
+AC_GHC_PKG_REQUIRE(old-time)
+AC_GHC_PKG_REQUIRE(temporary)
 
 #extra modules for monitoring daemon functionality; also needed for tests
 MONITORING_PKG=

